src/mmrelay/meshtastic_utils.py

```
@@ -743,8 +743,15 @@ def on_meshtastic_message(packet, interface):
     # Reaction handling (Meshtastic -> Matrix)
     # If replyId and emoji_flag are present and reactions are enabled, we relay as text reactions in Matrix
     if replyId and emoji_flag and interactions["reactions"]:
-        longname = get_longname(sender) or str(sender)
-        shortname = get_shortname(sender) or str(sender)
+        try:
+            longname = get_longname(sender) or str(sender)
+        except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)